src/libraries/System.Net.WebSockets/src/System/Net/WebSockets/WebSocketReceiveResult.cs

```diff
@@ -1,8 +1,6 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Diagnostics;
```

Comment:
I would abstain from removing unused namespaces from files in this PR as it is not related, and especially when it's the only change in the file.